eus_cddlib/euslisp/eus-cddlib-compiled.l

```diff
@@ -0,0 +1,2 @@
+(compiler::compile-file-if-src-newer (format nil "~A/euslisp/eus-cddlib" (ros::resolve-ros-path "package://eus_cddlib")))
```

Comment:
I think
`(let ((old-flag  ros::*compile-message*)) (setq ros::*compile-message* t) (load "package://eus_cddlib/euslisp/eus-cddlib.l")) (setq ros::*compile-message* old-flag)))` is fine.
https://github.com/jsk-ros-pkg/jsk_roseus/blob/b214284e24ad6287bf17545b9a73c541190fac1a/roseus/euslisp/roseus.l#L268

I'm also interested to see how much performance improvement can be achieved with the compiled code.